ocaml/xapi/quicktest_cbt.ml

```diff
@@ -81,13 +103,17 @@ let test ~session_id =
      * If not, skip that test, otherwise run it *)
     let run_test_suite ~session_id ~sR ~vDI =
       let sr_ops = (SR.get_allowed_operations ~session_id ~rpc:!rpc ~self:sR) in
-      [ (fun () -> enable_disable_cbt_test ~session_id ~vDI) ,
-        [ `vdi_enable_cbt ; `vdi_disable_cbt ]
+      [
+        (fun ~vDI -> vdi_data_destroy_test ~session_id ~vDI) ,
+        [ `vdi_enable_cbt ; `vdi_data_destroy ; `vdi_snapshot ]
       ]
       |> List.iter
-        (fun (test,list_vdi_ops) ->
+        (fun (test , list_vdi_ops) ->
            if List.for_all (fun vdi_op -> List.mem vdi_op sr_ops) list_vdi_ops
-           then test ()
+           then begin
+             debug cbt_test "Creating VDI...";
+             let vDI = make_vdi_from ~session_id ~sR in
```

Comment:
If I'm not mistaken, now we're creating a VDI for each test, so we can remove the old code at line 143, which used to create the global VDI share between all the tests?

Reply:
You are correct.